apps/src/templates/projects/ProjectWidget.jsx

```diff
@@ -1,6 +1,6 @@
 import React, {PropTypes} from 'react';
 import PersonalRecentProjects from './PersonalRecentProjects.jsx';
-import ContentContainer from '../ContentContainer.jsx';
+import {UnconnectedContentContainer as ContentContainer} from '../ContentContainer';
```

Comment:
It seems preferable to import the connected `ContentContainer` here, and remove the need to pass `isRtl` to it here.  That assumes this component is always used in a connected context though, which may not be the case.